go-controller/pkg/ovn/master.go

```diff
@@ -453,6 +453,13 @@ func (oc *DefaultNetworkController) deleteStaleNodeChassis(node *kapi.Node) erro
 				node.Name)
 			return fmt.Errorf("node %s is now with a new chassis ID. Its stale chassis entry is still in the SBDB", node.Name)
 		}
+		if err = libovsdbops.DeleteChassisTemplateRecord(oc.nbClient, []string{staleChassis}...); err != nil {
```

Comment:
It may be better to do this before we call `libovsdbops.DeleteChassisWithPredicate` in line 449. My thinking here is that if this fails, we will never be able to retry it since chassis will be gone and `oc.checkNodeChassisMismatch(node)` will not give us the stale stuff to remove from ChassisTemplate. wdyt?

Reply:
Makes sense, I changed it accordingly.